ant/org.eclipse.ant.core/src/org/eclipse/ant/internal/core/contentDescriber/AntBuildfileContentDescriber.java

```diff
@@ -59,7 +61,9 @@ private int checkCriteria(InputSource contents) throws IOException {
 		}
 		catch (SAXException e) {
 			// we may be handed any kind of contents... it is normal we fail to parse
-			AntCorePlugin.log(e);
+			// must not log an error - or test log listener will fail the test
+			IStatus status = new Status(IStatus.WARNING, AntCorePlugin.PI_ANTCORE, AntCorePlugin.INTERNAL_ERROR, "Error logged from Ant Core: ", e); //$NON-NLS-1$
```

Comment:
I would suggest to have the message reflect that it is "only" a warning and not an error to have consistent message and severity. You could also add a specific error code for invalid XML like it is already present for malformed URL and the like rather than passing the generic `INTERNAL_ERROR` because this is no real error.

Reply:
nitpicky: Log entries (for ANT builds) seem to be provided as NLS strings via properties files that may be localized. Maybe this message should be provided as proper NLS as well?

Reply:
that's just a copy of org.eclipse.ant.core.AntCorePlugin.log(Throwable) modified to IStatus.WARNING